src/lib/api.js

```diff
@@ -87,6 +87,15 @@ const api = {
     });
   },
 
+  /**
+   * "Ping" request to the API to see if it's alive. The request
+   * is made to a "lightweight" endpoint that only returns a HTTP 302,
+   * and an empty promise is returned instead.
+   */
+  ping: async () => {
+    await request.get(`${environment.apiUrl}/medic`);
+  },
```

Comment:
If we move the logic of trying and catching the request to the `lib/api` file, we get a few advantages: 
1. We keep the main file smaller and easier to follow. 
2. We don't have a log saying that we're checking some url when using archive mode, when we actually aren't. Besides being misleading, it can be also confusing. 
3. We don't have an empty function in the archiving api, rather one that always return true, which is easier to understand. 

```suggestion
  available: async () => {
    try {
      log.info(`Checking that ${environment.apiUrl}} is available...`);
      return await request.get(`${environment.apiUrl}/medic`);
    } catch (err) {
      log.error(`Failed to get a response from ${environment.apiUrl}. Maybe you entered the wrong URL, wrong port or the instance is not started? Please check and try again.`);
    }
  },
```